sdk/spring-sdk/src/test/java/kalix/springsdk/testmodels/eventsourcedentity/EventSourcedEntitiesTestModels.java

```diff
@@ -74,6 +78,49 @@ private Integer privateMethodSimilarSignature(Integer event) {
     }
   }
 
+  @Entity(entityKey = "id", entityType = "counter")
+  @RequestMapping("/eventsourced/{id}")
+  public static class WellAnnotatedESEntityWithJWT extends EventSourcedEntity<Integer> {
+
+    @GetMapping("/int/{number}")
+    @Subscribe.ValueEntity(UserEntity.class)
+    @JWT.JwtMethodOptions(
+        validate = {JwtMethodOptions.JwtMethodMode.BEARER_TOKEN},
+        sign = {JwtMethodOptions.JwtMethodMode.MESSAGE},
+        bearerTokenIssuer = {"a", "b"})
+    public Integer getInteger(@PathVariable Integer number) {
+      return number;
+    }
+
+    @PostMapping("/changeInt/{number}")
+    @Subscribe.ValueEntity(UserEntity.class)
+    @JWT.JwtMethodOptions(
+        validate = {JwtMethodOptions.JwtMethodMode.BEARER_TOKEN},
+        sign = {JwtMethodOptions.JwtMethodMode.MESSAGE},
+        bearerTokenIssuer = {"a", "b"})
+    public Integer changeInteger(@PathVariable Integer number) {
+      return number;
+    }
+
+    @EventHandler
+    public Integer receiveStringEvent(String event) {
+      return 0;
+    }
+
+    @EventHandler
+    public Integer receivedIntegerEvent(Integer event) {
+      return 0;
+    }
+
+    public Integer publicMethodSimilarSignature(Integer event) {
+      return 0;
+    }
+
+    private Integer privateMethodSimilarSignature(Integer event) {
+      return 0;
+    }
```

Comment:
I don't get that change. Were you testing out something else and committed it by accident?

Reply:
removed